pubspec.yaml

```diff
@@ -8,10 +8,10 @@ documentation: https://github.com/miquido/lint_quido
 
 environment:
   sdk: ">=2.19.2 <3.0.0"
```

Comment:
I would also recommend to update min constraint for dart version matching minimal used in latest flutter sdk. Will it be 3.0?